sites/platform/src/dedicated-environments/security-monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Security and monitoring"
+weight: -17
+sidebarTitle: "Security and monitoring"
+layout: single
+description:  "Security is handled in a similar way for both Dedicated Gen 2 and Dedicated Gen 3 projects, with strict procedures that are followed to handle incidents."
+---
+
+We use a combination of four trusted methods to ensure your site is secure and running at all times: 
+
+- [Project isolation](#project-isolation)
+- [Encryption](#encryption)
+- [Performance monitoring](#performance-monitoring)
+- [Security incident monitoring](#security-incident-monitoring)
+
+{{% description %}}
+
+### Project isolation 
+
+All Dedicated clusters are single-tenant. The [three hosts](https://docs.platform.sh/dedicated-gen-2/overview.html) are exclusively used by a single customer and each cluster is launched into its own isolated network (VPC on AWS, equivalent on other providers). The network is behind a firewall for incoming connections. Only ports 22 (SSH), 80 (HTTP), and 443 (HTTPS), and 2221 (SFTP) are opened to incoming traffic.
+
+There are no exceptions for this rule, so any incoming web service requests, ETL jobs, or otherwise need to transact over one of these protocols. Outgoing TCP traffic isn’t behind a firewall. Outgoing UDP traffic is disallowed. For containers to be allowed to connect to each other, the following requirement must be met:
+
+-   The containers must live in the same environment
+-   You need to define an explicit relationship between the containers in your [app configuration](https://docs.platform.sh/create-apps/app-reference/single-runtime-image.html#relationships)
+
+All Dedicated projects are isolated and their data is fully encrypted. Should a security breach occur, Platform.sh follows a strict [security incident handling** procedure](https://docs.platform.sh/dedicated-gen-3/security.html#security-incident-handling-procedure) to deal with the issue as promptly and efficiently as possible.
+
+### Encryption 
+
+| AWS | GCP | Azure |
```

Comment:
I think the GCP and Azure headers are flipped? The description below GCP seems like it is the description for Azure.

Reply:
I think you're right